LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 Dianomic Systems
+   Copyright 2018 Dianomic Systems Inc
```

Comment:
`2019`?

Reply:
We are only adding the missing Inc to the company name, the year should remain unchanged.